perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -210,6 +210,21 @@ sub body {
     return $self->from_body->name;
 }
 
+=head2 area_ids
+
+Returns the areas a user has been assigned to as an array
+
+=cut
+
+sub area_ids {
+    my $self = shift;
+    return [] if !defined $self->areas;
+    my @areas = split(/,/, $self->areas);
+    @areas = grep { $_ ne '' } @areas;
```

Comment:
This could be written as `@areas = grep length, @areas;`
Better, it could be combined with the previous line as `my @areas = grep length, split(/,/, $self->areas);`.